xml/depl_nodes.xml

```diff
@@ -4907,84 +4901,14 @@ openstack router add subnet <replaceable>router2</replaceable> priv-net-sub</scr
-    <term>SSL Support: Protocol
-        </term>
-    <listitem>
-     <para>
-      With the default value <guimenu>HTTP</guimenu> enabled, public
-      communication is not be encrypted. Choose <guimenu>HTTPS</guimenu> to use
-      SSL for encryption. See <xref linkend="sec-depl-req-ssl"/> for background
-      information and <xref linkend="sec-depl-inst-nodes-post-ssl"/> for
-      installation instructions. The following additional configuration options
-      will become available when choosing <guimenu>HTTPS</guimenu>:
-     </para>
-     <variablelist>
-      <varlistentry>
-       <term><guimenu>Generate (self-signed) certificates</guimenu>
-       </term>
-       <listitem>
-        <para>
-         When set to <literal>true</literal>, self-signed certificates are
-         automatically generated and copied to the correct locations. This
-         setting is for testing purposes only and should never be used in
-         production environments!
-        </para>
-       </listitem>
-      </varlistentry>
-      <varlistentry>
-       <term><guimenu>SSL Certificate File</guimenu> / <guimenu>SSL (Private) Key
-        File</guimenu>
-       </term>
-       <listitem>
-        <para>
-         Location of the certificate key pair files.
-        </para>
-       </listitem>
-      </varlistentry>
-      <varlistentry>
-       <term><guimenu>SSL Certificate is insecure</guimenu>
-       </term>
-       <listitem>
-        <para>
-         Set this option to <literal>true</literal> when using self-signed
-         certificates to disable certificate checks. This setting is for
-         testing purposes only and should never be used in production
-         environments!
-        </para>
-       </listitem>
-      </varlistentry>
-      <varlistentry>
-       <term><guimenu>SSL CA Certificates File</guimenu>
-       </term>
-       <listitem>
-        <para>
-         Specify the absolute path to the CA certificate. This field is
-         mandatory, and leaving it blank will cause the &barcl; to fail. To fix
-         this issue, you have to provide the absolute path to the CA
-         certificate, restart the <systemitem>apache2</systemitem> service, and
-         re-deploy the &barcl;.
-        </para>
-        <para>
-         When the certificate is not already trusted by the pre-installed list
-         of trusted root certificate authorities, you need to provide a
-         certificate bundle that includes the root and all intermediate CAs.
-        </para>
-       </listitem>
-      </varlistentry>
-     </variablelist>
-    </listitem>
-   </varlistentry>
-  </variablelist>
 
   <para>
-   The &o_meter; component consists of five different roles:
+   The &o_meter; component consists of four different roles:
   </para>
 
   <variablelist>
    <varlistentry>
-    <term><guimenu>ceilometer-server</guimenu>
+    <term><guimenu>ceilometer-server: The notification agent</guimenu>
```

Comment:
"The notification agent." should replace the `<para>` content below. The whole "API server" description is not valid anymore and there's not much to say about ceilometer-server besides the notification agent :o)